docs/source/openmw-cs/tour.rst

```diff
@@ -3,8 +3,8 @@ A Tour through OpenMW CS: making a magic ring
 
 In this first chapter we will create a mod that adds a new ring with a simple
 enchantment to the game. The ring will give its wearer a permanent Night Vision
-effect while being worn. You don't need prior knowledge about modding
-Morrowind, but you should be familiar with the game itself. There will be no
+effect while being worn. You do not need prior Morrowind modding experience, 
```

Comment:
You pull apart a contraction don't->do not, but you left prior?  Prior is a pompous term when previous would do just fine. :)